src/interfaces/modular/GaussianProcess_includes.i

```diff
@@ -25,7 +25,7 @@
 
  #include <shogun/machine/gp/InferenceMethod.h>
  #include <shogun/machine/gp/LaplacianInferenceBase.h>
- #include <shogun/machine/gp/FITCInferenceBase.h>
+ #include <shogun/machine/gp/SparseInferenceBase.h>
```

Comment:
I guess the motivation for this renaming is that we now not only have FITC inference but many sparse approximations?